README.md

```diff
@@ -251,6 +260,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [hq.cr](https://github.com/maiha/hq.cr) - Simple wrapper for crystal-xml
  * [modest](https://github.com/kostya/modest) - CSS selectors for HTML5 Parser myhtml
  * [myhtml](https://github.com/kostya/myhtml) - Fast HTML5 Parser
+ * [XML](https://crystal-lang.org/api/XML.html) - parsing and generating XML documents (Crystal stdlib)
```

Comment:
Are we accepting duplicated content?

Reply:
Well it kind of fits in both places... XML is a data format and it's obviously used for parsing HTML/XML.
Maybe the structure is wrong, but if we have these too topics, XML should be present in both.

Reply:
***Data Formats*** is broad enough to accept both parser and generators.

Reply:
We shouldn't suplicate libraries.

Reply:
@hugoabonizio Yeah, but if we put it there you wouldn't find it in * HTML/XML parsing* where you'd certainly expect this.

Reply:
@straight-shoota users are able to find it using search in a browser. Let's not  duplicate libraries.

Reply:
So... which one should go?

Reply:
Please leave the one in another section (Data Formats) and remove this line.